defaults/main.yml

```diff
@@ -5,6 +5,7 @@ kube_config_dir: "/etc/kubernetes"
 kube_cert_dir: "/etc/kubernetes/ssl"
 kube_manifests_dir: "/etc/kubernetes/manifests"
 kubernetes_service_addresses: "172.21.0.0/16"
+kubernetes_nodeport_range: "20000-32767"
```

Comment:
Can we keep this to the default range? Not every cluster would want so many ports.